lib/MusicBrainz/Server.pm

```diff
@@ -59,7 +59,6 @@ __PACKAGE__->config(
             'format_wikitext' => \&MusicBrainz::Server::Filters::format_wikitext,
             'format_editnote' => \&MusicBrainz::Server::Filters::format_editnote,
             'format_setlist' => \&MusicBrainz::Server::Filters::format_setlist,
-            'uri_decode' => \&MusicBrainz::Server::Filters::uri_decode,
```

Comment:
No longer needed. Decoding is mostly handled by `URI::as_iri`, plus some custom code in our URL class.